lib/zeus/api_client/version.rb

```diff
@@ -15,6 +15,6 @@
 module Zeus
   class APIClient
     # package version number
-    VERSION = "0.1.3"
+    VERSION = '0.1.3'.freeze
```

Comment:
@victorfonsec4 
BTW, we might update the version and abandon the existing versions later.